### ISSUE TITLE
Note about fast convergence in discrete LunarLander

### ISSUE THREAD

**Issue by ariel415el** [shivaverma/OpenAIGym#3]
Apparently the secret to this fast convergence in the discrete Lunar lander is doing 2 SGD steps at each environment step.
Keras Model.fit routine has a default batchsize of 32
https://github.com/shivaverma/OpenAIGym/blob/fad41cfb3dfdb08420895e9c959f418c2bfc52a9/lunar-lander/discrete/lunar_lander.py#L78
and since the script loads into it 64 samples (batch-size is 64 in the script) and trains for 1 epoch it does to gradient steps
It was driving me crazy: I tried to reproduce you results with pytorch with the same MLP model and achieved convergence only in almost 3 times more episodes.

Just mentioning

**Comment by shivaverma** [comment]
@ariel415el  I see, never paid attention to that. Ideally, the batch size should be 64 only at the time of fitting data.

You mean you achieved convergence in 3 times fewer episodes or more?

**Comment by ariel415el** [comment]
3 times more episodes. Yours is very fast.
